### PR TITLE
Updates to order of operations and accurate steps in the readme for creating a project

### PR DESCRIPTION
**Pull request by jordan-pieces**
Specific updates following usage by a contributor (sean) via Discord

**Description**
Goal is to make the dev experience outlined in the readme more accurate and easier to follow when first setting up an environment from scratch. Will also include some other updates as well with new endpoint examples. 

Also moving forward a goal of ours is to keep this readme and the readme on the npm package in sync with each other excluding some minor header details.

- [x] update order of ops and steps for creating a project from seans research 
- [x] update headers to reflect API calls instead of stoplight naming convention 
- [x] add additional examples from other projects and docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,61 +118,31 @@ import * as pieces from '@pieces.app/pieces-os-client'
 > **If you are having issues with your installation, it is likely due to a conflict in Typescript versions - `npm uninstall typescript` - then go back and perform all other npm installations before reinstalling typescript again**.
 
 
-### Creating the base of your project
-Let's get started with the base of your new React project where we will learn about manipulating Pieces OS and creating our own assets locally on device.
+## Creating a Project from Scratch
+Let's get started with the base of your new React project where we will learn about manipulating Pieces OS and creating our own assets locally on device. First we will create a directory for you project along with some dependancy setup:
```
Comment (CBID2) [inline README.md:122]: [spelling] Just a few typos
```suggestion
Let's get started with the base of your new React project! We will do this by learning more about manipulating Pieces OS and creating our own assets locally on device. First, we will create a directory for you project along with some dependancy setup:
```

**Comment by seantiz** [comment]
Just commenting to get on this

File: README.md [hunk 2]
```diff
@@ -118,61 +118,31 @@ import * as pieces from '@pieces.app/pieces-os-client'
...
-    "jsx": "react"  
-  },  
-  "exclude": ["node_modules"]  
-}
-```
-
-### Install Package Dependencies with npm
+1. Create a directory for your project called <copilot-project> (or the name of your choice)
+2. Open up a Terminal at the root of that directory, and run `npm uninstall typescript` to ensure you are starting fresh
+   - (optional) run `nvm install --lts` to check your node version
```
Comment (CBID2) [inline README.md:126]:
```suggestion
   - (optional) run `nvm install --lts` to check your  version of Node
```

File: README.md [hunk 3]
```diff
@@ -118,61 +118,31 @@ import * as pieces from '@pieces.app/pieces-os-client'
...
-
-### Install Package Dependencies with npm
+1. Create a directory for your project called <copilot-project> (or the name of your choice)
+2. Open up a Terminal at the root of that directory, and run `npm uninstall typescript` to ensure you are starting fresh
+   - (optional) run `nvm install --lts` to check your node version 
+3. From that same terminal window, use the `prefix` parameter in your install command to install your dependencies: `npm install --prefix ./ @pieces.app/pieces-os-copilot`
 
-#### Need this if you are going to work with ts files
+4. Install Dependencies with NPM with these three commands:
 ```bash
```
Comment (CBID2) [inline README.md:130]: [question]: It looks like 2 commands. What's the 3rd one?

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @jordan-pieces. I mostly gave feedback on grammar errors.

**Review by jordan-pieces — COMMENTED** [review]
👍

**Pull request merged**